super_team_picker/index.js

```diff
@@ -0,0 +1,44 @@
+//Express
```

Comment:
having unnecessary comments like this adds noise to the code. people who are reviewing your code should know by looking at it. if you have these minor comments in real practice, whoever reviews your code will ask you to remove these.